classification_projects/loan_default_prediction/src/feature_engineering/nodes.py

```diff
@@ -11,10 +11,10 @@ def encode_feature_values(col: pd.Series, encode_dict: Dict[str, int]) -> [pd.Se
 
 
 def sum_features(feat_to_sum: List[pd.Series], fill_val: Any = None) -> [pd.Series, int]:
-    res_col = pd.Series([])
+    res_col = feat_to_sum[0].copy()
```

Comment:
I don't understand why the result column is initialized as the first series in the array of series passed to the function in the parameters if this value will be added later anyway. Could you explain what was your thinking here?